web/src/scripts/webflow/webflowSync.test.js

```diff
@@ -261,6 +263,12 @@ jest.mock("fs", () => {
   };
 });
 
+jest.mock("./helpers2.mjs", () => {
+  return {
+    wait: jest.fn(),
+  };
+});
+
 describe("webflow syncing", () => {
```

Comment:
It would probably be helpful to add tests that confirm that `catchRateLimitErrorAndRetry` is called if the API does respond with something other than `200`. You can take or leave this now, but in a re-write this should for sure be covered.

Reply:
Isn't that the second test I wrote? "should retry function after waiting if rate limit error occurs" or did you mean something else?

Reply:
Either way I'm going to deal with it during a Friday as a sand